src/main/kotlin/com/ampnet/web3provider/repository/ProviderRedisRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.ampnet.web3provider.repository
+
+import java.math.BigInteger
+
+interface ProviderRedisRepository {
+    fun getBalance(address: String, blockParameter: Any): BigInteger?
```

Comment:
Does `blockParameter` have to be `Any`? Can you just use `String` instead?

Reply:
From the eth_wiki docs: `QUANTITY|TAG - integer block number, or the string "latest", "earliest" or "pending", see the default block parameter.` https://eth.wiki/json-rpc/API
It can be either an integer or a string.

Reply:
But they all have String value:
```
The following options are possible for the defaultBlock parameter:

    HEX String - an integer block number
    String "earliest" for the earliest/genesis block
    String "latest" - for the latest mined block
    String "pending" - for the pending state/transactions
```

Reply:
Yeah, ok, I''l change the impl of my web3Service to accept only string, and map the hex to bigInteger.